parca-dev/parca-agent
# feat(snap): add config option for external labels

### Why?

Allows a user to define a set of extra labels to attach to all profile to be able to identify profiles recorded in different contexts for the same thing.

### What?

Adding a new `metadata-external-labels` config option to the snap packaging.

### How?

`snap set parca-agent metadata-external-labels="foo=bar"`

### Test Plan

1. Install parca-agent snap on a test system
2. Configure a set of extra labels with `snap set parca-agent metadata-external-labels="foo=bar;bla=blup"`
3. Let the agent record some profiles and verify the labels are attached
